db/migrate/20190516113220_add_indexes_to_recorded_at_in_patients_and_blood_pressures.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexesToRecordedAtInPatientsAndBloodPressures < ActiveRecord::Migration[5.1]
```

Comment:
Why do we need two migration files?

Reply:
We added this later. @kitallis was of the opinion that we should keep this separate. Don't remember the arguments right now.

Reply:
merged them, updated the name